### DISCUSSION TITLE
progress bar not working

### DISCUSSION THREAD

**Discussion by FehmiCan16** [google/model-viewer#4632] [Q&A]
Hi, when I upload and download a model via the model viewer editor the progress bar does not appear. There was no such problem before.

live demo here:

https://3d.motaperp.com/bigballon/

What could be the reason why the progress bar is not working and how can I solve it?

**Comment by elalish** [comment]
Looking at your link, I'd say the problem is not the lack of a progress bar, but the lack of a 3D model - you've got a 404. Try updating your `src` path.

**Comment by Ju17350** [comment]
Hi. I'm facing the same issue with a default project (example using their Astronaut.glb model). I kept everything set by default, "show Progress Bar" is clicked of course, and I have set a poster. That's it. Tested on multiple browsers.
It's my first step with model viewer.
Is there any trick to know?

Thanks in advance.
Julien
